evalai/login.py

```diff
@@ -6,18 +6,18 @@
 from evalai.utils.auth import get_user_auth_token_by_login
 from evalai.utils.config import AUTH_TOKEN_PATH, AUTH_TOKEN_DIR
 
+@click.command()
 
-@click.group(invoke_without_command=True)
-@click.pass_context
-def login(ctx):
-    """
-    Login to EvalAI and save token.
-    """
-    username = click.prompt("username", type=str, hide_input=False)
-    password = click.prompt("Enter password", type=str, hide_input=True)
-    token = get_user_auth_token_by_login(username, password)
-
-    if os.path.exists(AUTH_TOKEN_PATH):
+@click.option('--auth')
+@click.option('--host')
+def main(auth,host):
+    
+    
+    a=click.prompt("AUTHENTICATION TOKEN")
+    h=click.prompt("HOST ID             ")
```

Comment:
Add space before and after `=`. Also remove trailing spaces in the `click.prompt`